src/common/types/feed.js

```diff
@@ -88,7 +93,22 @@ type AppActivityItems = {
     total_count: number,
 };
 
-type FeedItem = Comment | Task | BoxItemVersion | AppActivityItem;
+type AnnotationActivity = {
+    created_at: string,
+    created_by: User,
+    description?: AnnotationReply,
```

Comment:
Replies are eventually meant to be more generic, so this should probably just be `Reply`.